node/config.go

```diff
@@ -413,5 +418,5 @@ func makeAccountManager(conf *Config) (*accounts.Manager, string, error) {
 			backends = append(backends, trezorhub)
 		}
 	}
-	return accounts.NewManager(backends...), ephemeral, nil
+	return accounts.NewManager(backends...), keydir, nil
```

Comment:
Take care with this! The return parameter is the path only for **ephemeral** stores that the node will delete on shutting down!